runtime-services-modeling/decisions-with-dmn.adoc

```diff
@@ -704,6 +701,7 @@ Some boxed expressions are more complex with greater detail and calculation, suc
 .Boxed function expression for passenger reassignment
 image::dmn-reassign-passenger.png[]
 
+// replace with https://raw.githubusercontent.com/kiegroup/drools/8.39.0.Final/kie-dmn/kie-dmn-core/src/test/resources/org/kie/dmn/core/0019-flight-rebooking.dmn ?
```

Comment:
I'm happy with this idea.

```
[source]
----
include::https://raw.githubusercontent.com/kiegroup/drools/8.39.0.Final/kie-dmn/kie-dmn-core/src/test/resources/org/kie/dmn/core/0019-flight-rebooking.dmn[]
----

Reply:
running `asciidoctor -vw -R . -D output \**/*.adoc`

getting:
![image](https://github.com/IBM/bamoe-docs/assets/8044780/853aafa2-68bc-4bf1-8f32-15a15da4895b)


is that ok, and it wil be correct in official BAMOE docs?